tests/e2e/daemonset-features/add-scc-openshift.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [[ "$(kubectl api-resources)" =~ "operator.openshift.io" ]]; then
```

Comment:
The same here `kubectl api-resources --api-group=operator.openshift.io  -o name`